blog.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Neuton" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+  <link rel="stylesheet" href="styles/styles.css">
+  <title>Blog | Adoptadog</title>
+</head>
+
+<body>
+  <header>
+    <div class="checkout">
```

Comment:
You'll want to figure out a better way to lay these out so they live next to each other. Some ideas, flex, grid or positioning